### ISSUE TITLE
False positive when multiplying with two constants

### ISSUE THREAD

**Issue by reillywatson** [charithe/durationcheck#1]
Sample code:
```go
package dur

import "time"

const TwoDays = 2 * 24 * time.Hour
```
Outputs: "dur.go:5:17: Multiplication of durations: `2 * 24 * time.Hour`"

**Comment by charithe** [comment]
Thanks! I have a fix for this and the other case you raised but I need to clean up the code a bit more before I am happy to release it.

**Comment by reillywatson** [comment]
Awesome! Thanks for the nice tool, I've seen this bug before and am happy to have it never happen again :)

**Comment by charithe** [comment]
I am glad that you find it useful and thanks for the bug report.

This issue has been fixed.
